.github/workflows/php-code-coverage.yml

```diff
@@ -5,6 +5,9 @@ on:
     paths:
       - '**.php'
 
+permissions:
+  pull-requests: write
```

Comment:
Is this needed because we need to be able to write a coverage report?

Reply:
I actually just saw this requirement here https://github.com/marocchino/sticky-pull-request-comment?tab=readme-ov-file#basic . And tried to add it to see if it started working again. But we will probably know only after merging this